jetty-core/jetty-http3/jetty-http3-tests/src/test/java/org/eclipse/jetty/http3/tests/ClientServerTest.java

```diff
@@ -58,7 +64,7 @@ public void testConnectTriggersSettingsFrame() throws Exception
     {
         CountDownLatch serverPrefaceLatch = new CountDownLatch(1);
         CountDownLatch serverSettingsLatch = new CountDownLatch(1);
-        start(new Session.Server.Listener()
+        start(true, new Session.Server.Listener()
```

Comment:
Why `true` here?

Reply:
Good catch, that's a refactoring mistake. Let me revert that.